docs/Changelog.md

```diff
@@ -5170,6 +5254,67 @@ opset_import {
 <dd>Constrain output types to bool, int32, int64, float16, float, double tensors.</dd>
 </dl>
 
+### <a name="While-1"></a>**While-1**</a>
+
+  While Loop. This loop has multiple termination conditions:
+  
+  1) Static trip count. Iteration count specified staically in the model graph.
+      Set by setting the static_trip_count attribute (optional)
+  2) Dynamic trip count. Iteration count specified at runtime. Set by specifying
+     the input M, and setting the attribute has_trip_count = true (optional)
+  3) Loop conditional net. A net that has a single boolean result that specifies
+     whether the loop should continue. Set by specifying the cond attribute.
+  
```

Comment:
What is the semantics of trip_count in a loop? Is it an additional condition along with loop-condition or an alternative for the loop-condition?

Reply:
It's an additional condition. The loop continues while both (just updated) of these conditions hold true.